extensions/amp-story/1.0/amp-story.js

```diff
@@ -452,11 +464,47 @@ export class AmpStory extends AMP.BaseElement {
       );
     }
 
+    this.applyDesktopAspectRatioAttribute_();
+
     if (this.maybeLoadStoryDevTools_()) {
       return;
     }
   }
 
+  /**
+   * Grab the desktop-aspect-ratio attribute, clamp the value
+   * between 1/2 and 3/4 aspect ratios, and apply it to the root element.
+   * @private
+   */
+  applyDesktopAspectRatioAttribute_() {
+    if (!this.element.hasAttribute('desktop-aspect-ratio')) {
+      return;
+    }
+
+    const desktopAspectRatio =
+      'clamp(' +
```

Comment:
Pls clamp in JS so that the styles don't need to re-calculate it over and over.

Reply:
Done.